app/controllers/DvrController.scala

```diff
@@ -100,7 +104,8 @@ class DvrController @Inject()(
                     authId: Long,
                     submissionId: String) = authenticated {
     implicit request =>
-      propertyLinks.getLink(authId).map {
+      val pLink = if(request.organisationAccount.isAgent) propertyLinks.getClientAssessments(submissionId) else propertyLinks.getOwnerAssessments(submissionId)
```

Comment:
just because the org is an agent doesnt mean they are acting as an agent.

Reply:
Does this apply to everywhere in PLF? I've made the changes where you have commented but this pattern is used in a couple of more places??

Reply:
what places? if you can show me we can go threw them and see.

Reply:
flag now passed

Reply:
This pattern is still used in controllers AppointAgentController, ClaimProperty, DashBoard, Assessments and RevokeAgentController. I can change these as well